Blog/views.py

```diff
@@ -146,6 +147,12 @@ class PostCreate(LoginRequiredMixin, generic.CreateView):
     def form_valid(self, form):
         form.instance.user = self.request.user
         form.instance.created = datetime.datetime.now()
+        same_titles_count = Post.objects.filter(title__iexact=form.instance.title).count()
```

Comment:
There is still a problem: 
- Create a blog post having the title `Foo-Bar` -> Works
- Create a blog post having the title  `Foo Bar` -> Crash

Both titles result in the same slug `foo-bar`.

Reply:
How could this be solved? Please describe a possible solution...

Reply:
I'm really sorry I overlooked that one..
I could solve this Issue by searching for slugs instead of titles

Reply:
No worries. 

Yes, I think that‘s the safest way.

Reply:
i think i fixed this issue now